ui/package.json

```diff
@@ -22,49 +15,46 @@
     "gen:src": "./scripts/generate-client.sh"
   },
   "dependencies": {
-    "@patternfly/patternfly": "^4.224.2",
-    "@patternfly/react-core": "^4.276.8",
-    "@patternfly/react-icons": "^4.93.6",
+    "@patternfly/patternfly": "^6.1.0",
+    "@patternfly/react-core": "^6.0.0",
+    "@patternfly/react-icons": "^6.0.0",
+    "@tanstack/react-query": "^5.66.9",
     "axios": "^0.21.4",
+    "eslint": "^7.11.0",
     "formik": "^2.4.2",
     "history": "^5.0.1",
     "lodash": "^4.17.21",
     "moment": "^2.29.4",
-    "namor": "^2.0.3",
-    "prop-types": "^15.8.1",
-    "react": "^17.0.1",
-    "react-dom": "^17.0.1",
-    "react-feather": "^2.0.9",
+    "random-words": "^2.0.1",
```

Comment:
`random-words` is a replacement for `namor` used to generate random cluster names. The latter used the node `crypto` module, and was making the build process more cumbersome after moving to `vite`.